CMakeLists.txt

```diff
@@ -716,6 +725,10 @@ target_link_libraries(rr
   brotli
 )
 
+if(NOT ANDROID)
```

Comment:
slightly cleaner to set a ZSTD flag up above and use that here, instead of duplicating the ANDROID check